EVEStandard/API/Dogma.cs

```diff
@@ -32,6 +31,15 @@ internal Dogma(string dataSource) : base(dataSource)
             return returnModelDTO<DogmaAttribute>(responseModel);
         }
 
+        public async Task<ESIModelDTO<DogmaDynamicItem>> GetDynamicItemInfoV1Async(long typeId, long itemId, string ifNoneMatch = null)
```

Comment:
typeId should be int

Reply:
Should it? In that case, it needs to be changed in few more places.

```Find all "typeId", Find Results 1, Current Project: EVEStandard\EVEStandard.csproj, ""
  C:\Projects\EVEStandard\EVEStandard\API\Dogma.cs(34):        public async Task<ESIModelDTO<DogmaDynamicItem>> GetDynamicItemInfoV1Async(long typeId, long itemId, string ifNoneMatch = null)  
  C:\Projects\EVEStandard\EVEStandard\API\Market.cs(28):        public async Task<ESIModelDTO<List<MarketOrder>>> ListOrdersInRegionV1Async(int page, int regionId, long? typeId = null, string orderType = OrderType.All, string ifNoneMatch = null)
  C:\Projects\EVEStandard\EVEStandard\API\Market.cs(48):        public async Task<ESIModelDTO<List<MarketRegionHistory>>> ListHistoricalMarketStatisticsInRegionV1Async(int page, int regionId, long typeId, string ifNoneMatch = null)
  C:\Projects\EVEStandard\EVEStandard\API\Market.cs(128):        public async Task<ESIModelDTO<List<long>>> ListTypeIdsRelevantToMarketV1Async(int page, int regionId, string ifNoneMatch = null)
  C:\Projects\EVEStandard\EVEStandard\API\UserInterface.cs(61):        public async Task OpenMarketDetailsV1Async(AuthDTO auth, long typeId)```